config.json

```diff
@@ -9,8 +9,25 @@
         "dumpLevel": "error"
     },
     "redis": {
-        "host": "127.0.0.1",
-        "port": 6379
+        "name": "scality-s3",
+        "password": "",        "sentinels": [
```

Comment:
weird formatting, maybe hit backspace one too many times?

Reply:
whoops that wasn't supposed to be staged